src/Button/Button.tsx

```diff
@@ -32,7 +32,7 @@ const Button: FC<Props> = forwardRef<HTMLButtonElement, Props>(
       iconEnd,
       iconStart,
       intent = 'brand',
-      paletteColor = '',
+      paletteColor,
```

Comment:
`''` isn't a valid `PaletteColor`, but `undefined` should result in the same behavior.